cmd/archive-access-api/main.go

```diff
@@ -57,15 +58,17 @@ func run() int {
 
 	// Command line parameter initialization.
 	var (
-		flagAddress string
-		flagArchive string
-		flagCache   uint64
-		flagLevel   string
+		flagAddress     string
+		flagArchive     string
+		flagCache       uint64
+		flagMetricsAddr string
+		flagLevel       string
 	)
 
 	pflag.StringVarP(&flagAddress, "address", "a", "127.0.0.1:9000", "address to serve Access API on")
 	pflag.StringVarP(&flagArchive, "archive", "d", "127.0.0.1:80", "host URL for Archive API endpoint")
 	pflag.StringVarP(&flagLevel, "level", "l", "info", "log output level")
+	pflag.StringVarP(&flagMetricsAddr, "metrics", "m", "", "address on which to expose metrics (no metrics are exposed when left empty)")
```

Comment:
Do we want to set a default of `127.0.0.1:8080`?

Reply:
sure